### PR TITLE
Bug - mitre attack no result when missing attack pattern

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] In Hold - (Reason for hold)
- [ ] Ready

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-40623)

## Description
image
image

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Ni-Knight** [comment]
Is it only looking in the "Enterprise" collection?

**Review by Ni-Knight — APPROVED** [review]

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py [hunk 1]
```diff
@@ -511,14 +524,17 @@ def attack_pattern_reputation_command(client, args):
 
     filter_by_type = [Filter('type', '=', 'attack-pattern')]
     mitre_data = get_mitre_data_by_filter(client, filter_by_type)
-
+
```
Comment (RotemAmit) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py:527]: Remove the extra spaces.

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2_test.py [hunk 2]
```diff
@@ -268,6 +268,51 @@ def test_attack_pattern_reputation_command(mocker):
     assert command_results[0].indicator.value == 'Abuse Elevation Control Mechanism'
     assert command_results[1].indicator.value == 'Active Scanning: Wordlist Scanning'
 
+def test_attack_pattern_reputation_without_answer_command(mocker):
+    """
+    Given:
+        One attach patter to retrive data on, that is not found in the collection
```
Comment (RotemAmit) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2_test.py:274]:
```suggestion
        One attack pattern to retrive data on, that is not found in the collection
```

File: Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2_test.py [hunk 3]
```diff
@@ -268,6 +268,51 @@ def test_attack_pattern_reputation_command(mocker):
...
+    command_results = attack_pattern_reputation_command('', args)
+
+    assert command_results
+    assert command_results.readable_output == "MITRE ATTACK Attack Patterns values: No Attack Patterns found for ['dummy attack pattern']."
+
+
+def test_get_mitre_value_from_id_without_answer_command(mocker):
+    """
+    Given:
+        One attach patter to retrive data on, that is not found in the collection
```
Comment (RotemAmit) [inline Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2_test.py:297]:
```suggestion
        One attack pattern to retrive data on, that is not found in the collection
```

**Review by RotemAmit — COMMENTED** [review]
Nice!

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_40.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+
+- Fixed an issue where the commands ***attack-pattern***, ***mitre-get-indicator-name*** did not return an entry when no attack pattern data was found.
```
Comment (dantavori) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_1_40.md:6]:
```suggestion
- Fixed an issue where the ***attack-pattern*** and the ***mitre-get-indicator-name*** commands did not return an entry when no attack pattern data was found.
```

**Review by dantavori — APPROVED** [review]

**Pull request merged**